js/scripts.js

```diff
@@ -2,4 +2,4 @@ var withButtonClass = document.getElementsByClassName('button');
 
 for (var i = 0; i < withButtonClass.lenght; i++); {
 	alert(withButtonClass.innertext);
```

Comment:
you should call every element of array withButtonClass. withButtonClass is an Array :) try use var `i`